.gitlab/tests.yml

```diff
@@ -1,5 +1,8 @@
 variables:
   RIOT_RUN_CMD: riot -P -v run --exitfirst --pass-env -s
+  _DD_CIVISIBILITY_USE_PYTEST_V2: "true"
+  _CI_DD_ENV: "itrtest"
```

Comment:
This will be removed prior to merging (so this conversation should not be resolved without removing this line)
```suggestion
```